data/configs/config_new.json

```diff
@@ -10,7 +10,7 @@
   ],
   "settings": {
     "searchBar": true,
-    "searchUrl": "https://google.com/search?q=",
+    "searchUrl": "https://duckduckgo.com/?q=",
```

Comment:
As well as this. is it needed after 'yarn build'?

Reply:
Yes it is

Reply:
![image](https://user-images.githubusercontent.com/71191962/168486686-92a0d029-ca51-4da0-80fb-d9e37e3f1c58.png)

Reply:
not the edits the whole file as well os the others in /config